.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Can we not do this? Mostly for self-filed issues, we might not want to go through the whole template.

Reply:
Resolved in https://github.com/vmware/terraform-provider-nsxt/commit/50fcb7ad1539140b4f6f4a14864b57ee36196e59.